src/locales/de_DE.json

```diff
@@ -193,30 +210,35 @@
 "label.action.migrate.router.processing": "Router wird migriert....",
 "label.action.migrate.systemvm": "System-VM migrieren",
 "label.action.migrate.systemvm.processing": "System-VM wird migriert....",
-"label.action.project.add.account": "Konto zu Projekt hinzuf\u00fcgen",
+"label.action.project.add.account": "Konto zu Projekt hinzufügen",
 "label.action.reboot.instance": "Instanz neu starten",
 "label.action.reboot.instance.processing": "Instanz wird neu gebootet...",
 "label.action.reboot.router": "Router neu starten",
 "label.action.reboot.router.processing": "Router wird neu gebootet....",
 "label.action.reboot.systemvm": "System-VM neu starten",
 "label.action.reboot.systemvm.processing": "System-VM wird neu gebootet....",
 "label.action.recover.volume": "Recover Volume",
-"label.action.recurring.snapshot": "Wiederkehrende Schnappsch\u00fcsse",
+"label.action.recurring.snapshot": "Wiederkehrende Schnappschüsse",
 "label.action.register.iso": "ISO registrieren",
+"label.action.register.ncc": "NCC registrieren",
 "label.action.register.template": "Vorlage von URL registrieren",
-"label.action.release.ip": "IP losl\u00f6sen",
-"label.action.release.ip.processing": "IP wird losgel\u00f6st....",
+"label.action.release.ip": "IP loslösen",
+"label.action.release.ip.processing": "IP wird losgelöst....",
 "label.action.remove.host": "Host entfernen",
 "label.action.remove.host.processing": "Host wird entfernt....",
-"label.action.reset.password": "Passwort zur\u00fccksetzen",
-"label.action.reset.password.processing": "Passwort wird zur\u00fcckgesetzt....",
-"label.action.resize.volume": "Volumengr\u00f6\u00dfe \u00e4ndern",
-"label.action.resize.volume.processing": "Volumengr\u00f6\u00dfe wird ge\u00e4ndert....",
+"label.action.remove.vm": "Freigabe der VM",
+"label.action.reset.password": "Passwort zurücksetzen",
+"label.action.reset.password.processing": "Passwort wird zurückgesetzt....",
+"label.action.resize.volume": "Volumengröße ändern",
+"label.action.resize.volume.processing": "Volumengröße wird geändert....",
 "label.action.resource.limits": "Ressourcenlimits",
 "label.action.restore.instance": "Instanz wiederherstellen",
 "label.action.restore.instance.processing": "Instanz wird wiederhergestellt....",
-"label.action.revert.snapshot": "Auf Schnappschuss zur\u00fcckkehren",
-"label.action.revert.snapshot.processing": "Es wird auf den Schnappschuss zur\u00fcckgekehrt...",
+"label.action.revert.snapshot": "Auf Schnappschuss zurückkehren",
+"label.action.revert.snapshot.processing": "Es wird auf den Schnappschuss zurückgekehrt...",
+"label.action.router.health.checks": "Gesundheitsüberprüfungen des Routers anfordern",
+"label.action.run.diagnostics": "Diagnose starten",
+"label.action.secure.host": "Beschaffe Sicherheitsschlüssel des Hosts",
```

Comment:
Beschaffen???

Reply:
@svenvogel Can you help me check it. Thanks